vsintegration/src/FSharp.Editor/CodeFixes/AddOpenCodeFixProvider.fs

```diff
@@ -183,8 +188,8 @@ type internal AddOpenCodeFixProvider [<ImportingConstructor>] (assemblyContentPr
 
                         let entities =
                             assemblyContentProvider.GetAllEntitiesInProjectAndReferencedAssemblies checkResults
-                            |> Array.collect (fun s ->
-                                [|
+                            |> List.collect (fun s ->
```

Comment:
This didn't build for me, replaced it with `Seq.collect`.

Reply:
Because it was changed meanwhile. Needs to be Array.collect

Reply:
Wouldn't `seq` be better here anyway?

If this is used at all then it goes straight into `Seq.map`.